test/v2/models/response/retrieveBalanceAndTransactions/FinancialDetailsItemSpec.scala

```diff
@@ -42,70 +42,127 @@ class FinancialDetailsItemSpec extends UnitSpec with FinancialDetailsItemFixture
+        }
 
-          Seq("0" -> "No Reallocation").foreach((doTest _).tupled)
+        "convert empty string to false" in {
+          json("").as[FinancialDetailsItem] shouldBe model(false)
         }
 
-        "leave absent if present and no mapping is defined" in { // FIXME is this right?
-          json("UNKNOWN").as[FinancialDetailsItem] shouldBe model(None)
+        "convert absent field to false" in {
+          JsObject.empty.as[FinancialDetailsItem] shouldBe model(false)
+        }
+      }
+
+      "converting clearingLock to isEstimatedChargeOnHold" must {
+        def json(value: String): JsValue                = Json.parse(s"""{ "clearingLock": "$value" }""")
+        def model(value: Boolean): FinancialDetailsItem = financialDetailsItemModelEmpty.copy(isEstimatedChargeOnHold = value)
+
+        "convert non-empty string to true" in {
+          json("ANYTHING").as[FinancialDetailsItem] shouldBe model(true)
+        }
+
+        "convert empty string to false" in {
+          json("").as[FinancialDetailsItem] shouldBe model(false)
+        }
+
+        "convert absent field to false" in {
+          JsObject.empty.as[FinancialDetailsItem] shouldBe model(false)
+        }
+      }
+
+      "converting interestLock to isInterestAccrualOnHold" must {
+        def json(value: String): JsValue                = Json.parse(s"""{ "interestLock": "$value" }""")
+        def model(value: Boolean): FinancialDetailsItem = financialDetailsItemModelEmpty.copy(isInterestAccrualOnHold = value)
+
+        "convert non-empty string to true" in {
+          json("ANYTHING").as[FinancialDetailsItem] shouldBe model(true)
+        }
+
+        "convert empty string to false" in {
+          json("").as[FinancialDetailsItem] shouldBe model(false)
+        }
+
+        "convert absent field to false" in {
+          JsObject.empty.as[FinancialDetailsItem] shouldBe model(false)
         }
       }
 
-      "converting statisticalDocument to isStatistical" must {
-        def json(value: String): JsValue                        = Json.parse(s"""{ "statisticalDocument": "$value" }""")
-        def model(value: Option[Boolean]): FinancialDetailsItem = financialDetailsItemModelEmpty.copy(isStatistical = value)
+      "converting dunningLock to isInterestChargeOnHold" must {
+        def json(value: String): JsValue = Json.parse(s"""{ "dunningLock": "$value" }""")
+
+        def model(value: Boolean): FinancialDetailsItem = financialDetailsItemModelEmpty.copy(isInterestChargeOnHold = value)
+
+        "convert non-empty string to true" in {
+          json("ANYTHING").as[FinancialDetailsItem] shouldBe model(true)
+        }
+
+        "convert empty string to false" in {
+          json("").as[FinancialDetailsItem] shouldBe model(false)
+        }
+
+        "convert absent field to false" in {
+          JsObject.empty.as[FinancialDetailsItem] shouldBe model(false)
+        }
+      }
+
+      "converting statisticalDocument to isChargeEstimate" must {
```

Comment:
Tests for converting statisticalDocument match tech spec. However tech spec conflicts with main business reqs - business reqs still say: 
"Default to false, when it is 'G' then set it as True.

If it is blank, then it is false again. 

If the backend sends a value other than 'G', then we should consider it as a Severe Error. "

Can I confirm the business reqs above have been superseded by the tech spec version?